pkg/codegen/pcl/binder_test.go

```diff
@@ -13,37 +13,48 @@ import (
 )
 
 func TestBindProgram(t *testing.T) {
-	files, err := ioutil.ReadDir(testdataPath)
+	testdata, err := ioutil.ReadDir(testdataPath)
```

Comment:
Thanks so much for this! This test was a no-op before your fix. Quite an omission.